pkgs/development/interpreters/evcxr/default.nix

```diff
@@ -54,6 +53,10 @@ evcxr.overrideAttrs (oldAttrs: {
           "--set" "CARGO_HOME" "${cargoHome packageNames}"
           "--prefix" "PATH" ":" "${lib.makeBinPath [rustc cargo]}"
         ];
+
+        passthru = {
+          cargoHome = cargoHome packageNames;
+        };
```

Comment:
This calls `cargoHome packageNames` twice (here and in the `makeWrapperArgs`). I'm not familiar with this: what does this passthru do?

Reply:
It doesn't do anything for this derivation. It turned out to be useful for a downstream project I'm working on to make the `CARGO_HOME` directory available. 

In my case, I wanted to build an environment with `evcxr.withPackages`, and also run `rust-analyzer` against the same environment.